src/commands/distribute/releases/edit.ts

```diff
@@ -29,27 +31,33 @@ export default class EditReleaseCommand extends AppCommand {
   public releaseId: string;
 
   @help("Release state: enabled or disabled")
-  @name("State")
-  @position(0)
-  @required
+  @shortName("s")
```

Comment:
While I agree this should not be a positional parameter (I didn't even know the CLI supported that), this change is breaking. People may have scripts running this operation. Since the command already includes many other named parameters, I doubt it is necessary to make it work? Let's leave this alone. What do you think @jp-andre ?

Reply:
Correct, we can't have a breaking change here.

Reply:
Note that I don't know if we can have an optional positional parameter, that will need to be investigated.

Reply:
@jp-andre @anywherepilot that's an issue. Which way is more acceptable: create new command such as ''patch" for example or change current one with breaking back compatibility?

Reply:
Or just 'editnotes' I think that is most clear way which can resolve all the issues with back compatibility and updating to empty notes

Reply:
I would try to see whether an optional positional parameter works. That will make this command a little weird to work with regarding that one parameter, but at least it still has a discoverable name and is extensible. That's important because there are many more things you can change about a release through the API, that we may want to add to the CLI in the future. The PUT call doesn't have that many options, but the PATCH one does. I don't know why we have that distinction, but to me it'd make sense so call both from this single CLI command.

@jp-andre are you in favor of one edit command that does everything, with one weird positional parameter (if this can be done), or would you rather have separate commands for edit-release-notes and then in the future edit-mandatory and edit-notify-testers, etc.?